.github/workflows/dev-checks.yml

```diff
@@ -90,7 +94,14 @@ jobs:
            echo "Branch name does not start with 'TS-' or 'revert-' or 'TDEVOPS-'"
            exit 1
          fi
-     
+      - name: Check double commit
```

Comment:
If this check is supposed to be conditional based on value of `check_double_commit` input, then, you need to split it into two. 
The login username check is MUST and always. It is not user controlled. The second part, which is to match the double commit string in title, etc.

Reply:
Looking into it there's no need for a conditional and this check must be followed in every case, so I'll just clear up the conditional